metric_tank/http.go

```diff
@@ -105,9 +105,9 @@ func Get(w http.ResponseWriter, req *http.Request, store Store, metaCache *MetaC
 		}
 
 		if consolidateBy == "" {
-			meta := metaCache.Get(id)
+			def, ok := defCache.Get(id)
 			consolidateBy = "avg"
-			if meta.targetType == "counter" {
+			if ok && def.TargetType == "counter" {
```

Comment:
The id's sent in the query to MT have come from ES. So it is an error if the metricDef is not in our cache.  Can we log the error here